beps/bep_0048.rst

```diff
@@ -0,0 +1,81 @@
+:BEP: 48
+:Title: Tracker Protocol Extension: Scrape
+:Version: $Revision$
+:Last-Modified: $Date$
+:Author:  Jimmy Zelinskie <jimmyzelinskie@gmail.com>
+:Status:  Draft
+:Type:    Standards Track
+:Content-Type: text/x-rst
+:Created: 08-Sep-2016
+:Post-History:
+
+
+Abstract
+--------
+
+The original BitTorrent Protocol Specification (BEP 003) [#BEP_003]_ defines an exchange between a client and a tracker referred to as an Announce.
+An Announce has two objectives: update a peer's presence in the Swarm and discover other peers in the Swarm.
```

Comment:
I don't think we should use such a closed definition for announces here, it's not the scope of the document. Just stating that scrapes are more scalable and do not signal participation in the swarm should be good enough.